### PR TITLE
Update Github action workflows to align with our compass conventions

### PR DESCRIPTION
**Pull request by junan**
## What happened

Updated the Github action workflows to align with our [compass conventions](https://nimblehq.co/compass/development/code-conventions/github-actions/)
 
## Insight

- Removed quote from the workflow name
- Add `name` attributes for each action to describe the action
- Add a space between actions
- Renamed "Tests" to "test" for the test template workflow name
 
## Proof Of Work

All tests should be passed

### CODE REVIEW

**Review by andyduong1920 — APPROVED** [review]

**Comment by andyduong1920** [comment]
@junan Rebase required 🙏

**Comment by olivierobert** [comment]
@junan Please rebase so that this can be merged in the upcoming release

File: priv/templates/nimble_template/.github/workflows/test.yml.eex [hunk 1]
```diff
@@ -31,16 +32,19 @@ jobs:
...
           access_token: ${{ github.token }}
 
-      - uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
 
-      - uses: erlef/setup-elixir@v1
+      - name: Setup Elixir
```
Comment (olivierobert) [inline priv/templates/nimble_template/.github/workflows/test.yml.eex:40]: To be a verb it should `Set up`. In one word `Setup` is a noun.

**Review by olivierobert — CHANGES_REQUESTED** [review]

**Review by longnd — APPROVED** [review]

**Review by bterone — APPROVED** [review]

**Comment by andyduong1920** [comment]
Please rebase this PR @junan  the test was fixed on #128 👍

**Comment by junan** [comment]
>  Please rebase this PR @junan the test was fixed on #128 👍

@andyduong1920 Test has not been fixed, after rebase it is still failing. To verify it, I created new branch a committed a small [commit](https://github.com/nimblehq/elixir-templates/commit/eb913628896cf40e3c1a7b3cf8d175a16625f64d), but still [fails](https://github.com/nimblehq/elixir-templates/runs/4405259299?check_suite_focus=true) 💭

**Comment by andyduong1920** [comment]
> > Please rebase this PR @junan the test was fixed on #128 👍
> 
> @andyduong1920 Test has not been fixed, after rebase it is still failing. To verify it, I created new branch a committed a small [commit](https://github.com/nimblehq/elixir-templates/commit/eb913628896cf40e3c1a7b3cf8d175a16625f64d), but still [fails](https://github.com/nimblehq/elixir-templates/runs/4405259299?check_suite_focus=true) 💭

It's another issue due to the upstream introduced by just today I guess @junan 😂 while the other issue was fixed in #128

File: .github/workflows/test_live_variant_on_standard_liveview_project.yml [hunk 2]
```diff
@@ -30,16 +31,20 @@ jobs:
...
 
-      - uses: erlef/setup-elixir@v1
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
         with:
           otp-version: ${{ env.OTP_VERSION }}
           elixir-version: ${{ env.ELIXIR_VERSION }}
 
-      - uses: actions/setup-node@v2.1.5
+
```
Comment (rosle) [inline .github/workflows/test_live_variant_on_standard_liveview_project.yml:45]:
```suggestion
```

**Pull request merged**